src/ui/settings-tab.ts

```
@@ -0,0 +1,746 @@
+  }
+
+  async display(): Promise<void> {
+    let { containerEl } = this;
+
+    containerEl.empty();
+
+    containerEl.createEl('h1', { text: 'Readwise Sync Configuration' });
+
+    const apiTokenFragment = document.createDocumentFragment();
+    apiTokenFragment.createEl('span', null, (spanEl) =>
+      spanEl.createEl('a', null, (aEl) => (aEl.innerText = aEl.href = 'https://readwise.io/access_token'))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)